### DISCUSSION TITLE
PrimeReact Package Size

### DISCUSSION THREAD

**Discussion by meriemMat** [primefaces/community#268] [PrimeReact] [answered]
Hello Everyone;
I've Noticed after migrating from PrimeReact v 8.7.3 to Version 10.0.3 that the bundle size rised considerably by arround 2mb(transfered over Network). is there a rise of the  package size between the two version??

**Comment by buraksaglam089** [comment, marked as answer]
Hello,

The recent major updates of PrimeReact has led to a size difference between the two versions. These major updates includes pass through and Unstyled mode. You can use these features with newest version.

Regards.

**Reply by meriemMat** [reply]
Thank You @buraksaglam089 for your answer and clarification 👍 
I Have some Issues when the main interface Shows , its slower so that i can see elements taking there places (Spliters.,tabs .....etc) 
am not sure why this is hapening after upgrade , and  what can be the prblem behind this .
we preserved our updated css files , am i missing some details here ?
![image](https://github.com/primefaces/community/assets/17379300/9516cbe5-3398-4e34-847e-df8b1c232d3b)

**Comment by meriemMat** [comment]
i am having this ui shwoing quiqly then the css is correctly set :
![Sans titre](https://github.com/primefaces/community/assets/17379300/0b357363-e7b3-4825-ad75-8cd2ea2bc409)
Any idea about the cause?

**Comment by mertsincan** [comment]
@nitrogenous can you assist with this question?

**Comment by nitrogenous** [comment]
We couldn't be able to reproduce the issue you mention. I am currently going to close this discussion. If your issue continues I kindly ask you to share more information with us. Warmest regards
